source/developer/oauth2-applications.rst

```diff
@@ -0,0 +1,80 @@
+OAuth 2.0 Applications
+===============
+
+OAuth 2.0 is a protocol that allows Mattermost to authorize API requests from external applications. The authorization lets 
+
+ - users with an account on a Mattermost server to sign in to third-party applications. You can find a `sample OAuth2 Client Application for Mattermost here <https://github.com/enahum/mattermost-oauth2-client-sample>`_ to test the functionality.
+
+ - a Mattermost server to authenticate requests to a third-party API. One popular application is Zapier integration which allows you to integrate more than 500 applications with Mattermost through OAuth 2.0. See our `Zapier documentation <https://docs.mattermost.com/integrations/zapier.html>`_ to learn more.
+
+This documentation includes the following details:
+
+ - `Register your application in Mattermost <https://docs.mattermost.com/developer/oauth-2-0-applications#register-your-application-in-mattermost>`_
+ - `Permissions to your application  <https://docs.mattermost.com/developer/oauth-2-0-applications#permissions-to-your-application>`_
```

Comment:
Is this supposed to be "Managing" or "Granting" permissions to your application.